src/bundle/Controller/ContentController.php

```diff
@@ -19,6 +19,10 @@
 use eZ\Publish\Core\Base\Exceptions\BadStateException;
 use eZ\Publish\Core\Helper\TranslationHelper;
 use eZ\Publish\SPI\Limitation\Target;
+use EzSystems\DateBasedPublisher\API\Repository\DateBasedHideServiceInterface;
```

Comment:
I know this is unfinished but just heads up - you can't use anything coming from EE packages. If you belive this could be shared then move those classes to AdminUI (or other package if you feel it would be more accurate) and refactor Date Based Publisher.